### PR TITLE
UI-Dashboard

### LINKED ISSUE #421: Dashboard Page
This issue is for the dashboard page.

### PR DESCRIPTION
**Pull request by kaimanasse15**
resolves #421 

This is the UI that I got from @BradleyCharles. It still has some error.
Link: https://www.figma.com/design/yQHLPUfwOwlAj9gyBMlA85/Dashboard?t=8iE6XkYoYnnoTecf-0

### CODE REVIEW

**Comment by tinpham5614** [comment]
Are you still working on this PR?
Is this the expected result?
image

**Comment by kaimanasse15** [comment]
I still need to discuss how the design should be done, but it looks okay so far. Are you the project manager for the NSC, @tinpham5614 ?

**Comment by brinkbrink** [comment]
Hi @kaimanasse15 what is this pull request trying to accomplish? I'm unsure of what to review. Thanks!

**Comment by brinkbrink** [comment]
c

> I still need to discuss how the design should be done, but it looks okay so far. Are you the project manager for the NSC, @tinpham5614 ?

You can reach out to @taylorpapke, @kaimanasse15. @tinpham5614 is the lead developer.

File: app/dashboard/page.tsx [hunk 1]
```diff
@@ -1,11 +1,76 @@
...
 
 const Dashboard = () => {
+  const [drawerOpen, setDrawerOpen] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(0);
+
+  const handleDrawerToggle = () => {
+    setDrawerOpen(!drawerOpen);
+  };
+
+  const handleListItemClick = (index) => {
```
Comment (taylorpapke) [inline app/dashboard/page.tsx:14]: For Line 14, a potential quick fix is the following:
Screenshot 2024-06-04 at 11 53 02 PM

**Review by taylorpapke — APPROVED** [review]
Looks good:
Screenshot 2024-06-04 at 11 51 05 PM

Just need to fix the error and conflict.

**Comment by taylorpapke** [comment]
> I still need to discuss how the design should be done, but it looks okay so far. Are you the project manager for the NSC, @tinpham5614 ?

@keiffer213 is the project manager now.

**Review by Seiyaroo — COMMENTED** [review]
@kaimanasse15  or @taylorpapke do we know if this is still being worked on or taken over?

**Comment by keiffer213** [comment]
@Seiyaroo It is not currently being worked on right now, but this will probably be open for someone to take over

**Comment by taylorpapke** [comment]
@Seiyaroo @keiffer213 This should be ready to merge. Just needs 1 more approval.

**Review by Seiyaroo — APPROVED** [review]
LGTM per Taylors approval

**Pull request merged**